shared/domains/chelonia/chelonia.js

```diff
@@ -260,6 +260,13 @@ sbp('sbp/selectors/register', {
     // calling this will make pubsub unsubscribe for events on `contractID`
     sbp('okTurtles.events/emit', CONTRACTS_MODIFIED, state.contracts)
   },
+  'chelonia/contractEventsBefore': async function (contractID: string, before: string, limit: number) {
+    if (limit <= 0) {
+      console.error('[chelonia] invalid params error: "limit" needs to be positive integer')
+      return
+    }
+    return await sbp('chelonia/private/out/eventsBefore', contractID, before || '', limit)
+  },
```

Comment:
I'd make a few suggestions here:

1. Rename to `'chelonia/eventsBefore'`
2. Remove `contractID` from the parameters
3. Delete the `'chelonia/private/out/eventsBefore'` selector and move its implementation into here, since this is essentially just a wrapper around that

Reply:
*edited above message to suggest rename to `'chelonia/out/eventsBefore'`.

Also, for consistency then, might be worth also moving `'chelonia/private/out/eventsSince'` into this file and renaming it to `'chelonia/out/eventsSince'`

Reply:
Got it.

Reply:
Got it.